config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: docker.io/apecloud/kubebench
-  newTag: "0.1.0"
+  newName: docker.io/guang999/kubebench
```

Comment:
Why does not use apecloud registry

Reply:
this is auto generate when I execute `make deploy` with my docker image. I will rollback it